### ISSUE TITLE
Agenda for sync meeting 1/22/21

### ISSUE THREAD

**Issue by tlively** [WebAssembly/simd#419]
The next meeting will be Friday, January 22 at 9:00AM - 10:00AM PST/ 6:00PM - 7:00PM CET. Please respond with agenda items you would like to discuss.

If this meeting doesn't already appear on your calendar, or you are a new attendee, please fill out this [form](https://forms.gle/9eB2ZYaziPEcTJabA) to attend.

Agenda so far:

 - Expectations about engine optimizations w.r.t. optimizing the users of comparisons
 - i64x2.all_true (#415)
 - Signed i64x2 comparisons (#412)
 - Unsigned i64x2 comparisons (#414)
 - i64x2.ne (#411)
 - i64x2.eq (#381)
 - i64x2.abs (#413)
 - Sign select (#124) if @rrwinterton is able to look into its Alder Lake performance

**Comment by penzn** [comment]
We had another item on agenda for today - the cost model. IMO it probably lower in priority to everything on the list, but can complement the optimization discussion.

Also, what about i64x2 min and max (#417, #418)?

**Comment by tlively** [comment]
Thanks, min and max were accidental omissions from my list. I've been putting off the cost model discussion for now because it doesn't block moving to phase 4. It's an important conversation to have, though. After the phase 4 vote I expect that there will be a good amount of additional optimization, documentation, and cleanup work to do on the tools side.

**Comment by Maratyszcza** [comment]
New instructions with experimental data to discuss:
- Population Count (#379)
- Extended Pairwise Addition (#380)
- Prefetch (#352)

**Comment by lars-t-hansen** [comment]
Given where we are wrt work load and looming phase 4 deadline, we might consider scheduling another meeting next week.

**Comment by tlively** [comment]
Thanks for a productive meeting, everyone! Here are the [notes](https://docs.google.com/document/d/1hmf-WSDbJWHX_zln-sXkttlgrRMcugRYxjjIeJngUGI/edit?usp=sharing).

We conducted five votes this meeting:

 1. We had consensus that in subsequent discussions about whether to include particular instructions, we should assume engines will be able to recognize and optimize mask operands that were produced by comparison operations.
 2. We had consensus that `i64x2.all_true` (#415) should be merged to the proposal.
 3. We had consensus that `i64x2.eq` (#381) should be merged to the proposal.
 4. We had consensus that `i64x2.ne` (#411) should be merged to the proposal.
 5. We did **not** have consensus that i64x2 signed comparisons should be merged to the proposal. They will not be added unless we get more data and revisit the discussion, but this is a low priority.

We also decided to meet again at the same time next week to discuss the agenda items we did not get to in this meeting.
